spec/takeaway_spec.rb

```diff
@@ -0,0 +1,62 @@
+require 'order'
+require 'menu'
+require 'pricecalculation'
+
+describe Order do
+  let(:subject) { Order.new }
+  let(:checkout) { double :checkout }
+
+  it 'shows menu including list of dishes and prices' do
+
+    expect(subject.read_menu).to eq subject.read_menu
+  end
+
+  it 'should allow you to select menu items and add them to order' do
```

Comment:
It would be nice to have these tests formatted into describe '#method' block.

Reply:
also correct